appveyor.yml

```diff
@@ -15,9 +15,15 @@ install:
   - rustc -vV
   - cargo -vV
 build_script:
-  - cargo build --features vulkan
+  - >
+    cargo build --all --features vulkan
+    --exclude gfx_window_glfw
+    --exclude gfx_device_metal
+    --exclude gfx_device_metalll
+    --exclude gfx_window_metal
 test_script:
-  - cargo test --all --features vulkan
-  - cargo test -p gfx -p gfx_core --features "mint serialize"
-  - if %COMPILER%==msvc cargo test --all --features dx11
-  - if %COMPILER%==msvc cargo test --all --features dx12
+  # re: gfx is gfx_render
+  - cargo test -p gfx -p gfx_core
```

Comment:
Well, if you ask me, I'd rather remove *all* those headers entirely. They are just cluttering the source.
Last time we discussed this with the gfx-rs core team (~3 years ago? lol), the main argument for keeping the comments was that Rust uses the same scheme. Hard to counter that  :(